### PR TITLE
[15.0][MIG] account_invoice_import

### PR DESCRIPTION
**Pull request by koenloodts**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I think this one should rely on the new EDI infrastructure that Odoo core includes. What do you think?

**Comment by tbanevicius** [comment]
Hi @koenloodts @pedrobaeza  . Is there anything I can do to help this MR to move on?

**Comment by bosd** [comment]
> I think this one should rely on the new EDI infrastructure that Odoo core includes. What do you think?

I think the same.
Maybe an deeper analysis is needed to see if this module is still needed? Some items can maybe be handled in core. Some dicts might need a build out. 

I'm currently working on the invoice2data invoice line functionality.
Currently struggling to get the data passed correctly trough this module.

For example:
Lines without taxes get mapped to a taxtag, lines with taxes are not mapped to a tax tag. 

Need some more time to do some more code diving. Maybe it is easier to bypass this module.

File: account_invoice_import/wizard/account_invoice_import.py [hunk 1]
```diff
@@ -0,0 +1,1456 @@
...
+        il_vals = {
+            "name": _("Adjustment"),
+            "quantity": sign,
+            "price_unit": diff_amount * sign,
+        }
+        # no taxes nor product on such a global adjustment line
+        if import_config["invoice_line_method"] == "nline_no_product":
+            il_vals["account_id"] = import_config["account"].id
+        elif import_config["invoice_line_method"] == "nline_static_product":
+            account = amlo.get_invoice_line_account(
```
Comment (bosd) [inline account_invoice_import/wizard/account_invoice_import.py:906]: Can you include a fix for:
https://github.com/OCA/edi/issues/572

It is a leftover of V14 migration.

**Pull request closed without merging**